src/gpu/intel/sycl/l0/utils.cpp

```diff
@@ -272,6 +287,115 @@ bool compare_ze_devices(const ::sycl::device &lhs, const ::sycl::device &rhs) {
     return lhs_ze_handle == rhs_ze_handle;
 }
 
+status_t get_device_ip(ze_device_handle_t device, uint32_t &ip_version) {
+    ze_device_properties_t deviceProps = {ZE_STRUCTURE_TYPE_DEVICE_PROPERTIES};
+    ze_device_ip_version_ext_t devicePropsIP
+            = {ZE_STRUCTURE_TYPE_DEVICE_IP_VERSION_EXT};
+    deviceProps.pNext = &devicePropsIP;
+    func_zeDeviceGetProperties(device, &deviceProps);
+    ip_version = devicePropsIP.ipVersion;
+    return status::success;
+}
+
+status_t get_lz_device_enabled_systolic_intel(
+        ze_device_handle_t device, bool &mayiusesystolic) {
```

Comment:
Nit, for consistency with the call args:
```suggestion
        ze_device_handle_t device, bool &mayiuse_systolic) {
```